lms/djangoapps/experiments/utils.py

```diff
@@ -74,20 +73,19 @@
 )
 # TODO END: Clean up as part of REV-1205 (End)
 
-# .. toggle_name: streak_celebration.discount_experiment_AA759
-# .. toggle_implementation: ExperimentWaffleFlag
+# .. toggle_name: streak_celebration.discount
+# .. toggle_implementation: WaffleFlag
 # .. toggle_default: False
-# .. toggle_description: This experiment flag enables an engagement discount incentive message.
+# .. toggle_description: This flag enables an engagement discount incentive message.
 # .. toggle_warnings: This flag depends on the streak celebration feature being enabled
 # .. toggle_use_cases: temporary
-# .. toggle_creation_date: 2021-05-05
-# .. toggle_target_removal_date: 2021-07-05
-# .. toggle_tickets: https://openedx.atlassian.net/browse/AA-759
-STREAK_DISCOUNT_EXPERIMENT_FLAG = ExperimentWaffleFlag(
-    LegacyWaffleFlagNamespace(name='streak_celebration'),
-    'discount_experiment_AA759',
-    __name__,
-    use_course_aware_bucketing=False
+# .. toggle_creation_date: 2021-08-26
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: https://openedx.atlassian.net/browse/AA-950
```

Comment:
Is this still tech debt? Should I convert all the LegacyWaffleFlags to WaffleFlags?

Reply:
hm yeah i have no idea why i used a LegacyWaffleFlag, i think you can change that to a WaffleFlag, just search the code for an example 
https://github.com/edx/edx-toggles/blob/6d1c566c95981348ef8de6c26a8c31e71ced2850/docs/how_to/documenting_new_feature_toggles.rst#refactor-legacywaffle-classes

Reply:
ok, converted all the LegacyWaffleFlags in the module to WaffleFlags